src/deepsparse/transformers/pipelines/question_answering.py

```diff
@@ -315,8 +315,22 @@ def route_input_to_bucket(
         :param pipelines: Different buckets to be used
         :return: The correct Pipeline object (or Bucket) to route input to
         """
-        current_seq_len = len(input_schema.question.split())
-
+        tokenizer = pipelines[0].tokenizer
+        context_tokens = tokenizer(
+            input_schema.context,
+            add_special_tokens=True,
+            return_tensors="np",
+            padding=False,
+            truncation=False,
+        )
+        question_tokens = tokenizer(
+            input_schema.question,
+            add_special_tokens=True,
+            return_tensors="np",
+            padding=False,
+            truncation=False,
+        )
+        current_seq_len = len(context_tokens) + len(question_tokens)
         for pipeline in pipelines:
             if pipeline.sequence_length > current_seq_len:
                 return pipeline
```

Comment:
looking at this again - this assumes that the pipelines are sorted by sequence length is this always true?